### ISSUE TITLE
Test: status bar accessibility

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#96265]
Refs: https://github.com/microsoft/vscode/issues/94677

- [x] win @dbaeumer
- [x] linux @bpasero

Complexity: 2

---

If you are on Windows setup NVDA screen reader.
If you are on Linux setup Orca latest from source (tested that it work on Ubuntu 18, ArchLinux, Fedora 31. Ubuntu 19 showed some issue) [docs](https://code.visualstudio.com/docs/editor/accessibility#_linux). Some [troubleshooting](https://github.com/microsoft/vscode/issues/91063)

Turn on your screen reader, open VS Code. For NVDA press `Insrt + End` and the status bar content should be read. Make sure that what is read makes sense and captures what you see in the status bar.
For Orca you can `orca -s` from the command line and configure the keybinding for the read status command. Trigger that command and make sure that the status bar gets read out.

In both cases no duplicated content should be read out, and what is being read should be concise and descriptive enough.

**Comment by isidorn** [comment]
For setting up Orca from master this comment has all the steps 
https://github.com/microsoft/vscode/issues/90446#issuecomment-585782600

**Comment by isidorn** [comment]
Since @bpasero is having issues setting up Orca.
@marcozehe might you know if VoiceOver has a command to focus the status bar? I searched and could not find one? 
@joanmarie is there an easier way to setup orca with vscode then to take one from Master? In Ben's VM he can not check off Canocical Packages in his Ubuntu 18 and thus can not install latest Orca.

**Comment by MarcoZehe** [comment]
No, there is no command to read the status bar. Status bars aren't as common on MacOS it seems.

**Comment by bpasero** [comment]
I could get Orca to run (from `master`) and verified that pressing the keybinding for reading status bar worked for me on Ubuntu 18 VM.

**Comment by isidorn** [comment]
Great thanks @bpasero
@joanmarie please ignore this thread then.
